windows/src/global/delphi/general/PackageInfo.pas

```diff
@@ -1923,6 +1930,7 @@ procedure TPackageKeyboardList.LoadXML(ARoot: IXMLNode);
     keyboard.Name := XmlVarToStr(AKeyboard.ChildValues[SXML_PackageKeyboard_Name]);
     keyboard.ID := XmlVarToStr(AKeyboard.ChildValues[SXML_PackageKeyboard_ID]);
     keyboard.Version := XmlVarToStr(AKeyboard.ChildValues[SXML_PackageKeyboard_Version]);
+    keyboard.RTL := ANode.ChildNodes.IndexOf(SXML_PackageKeyboard_RTL) >= 0;
```

Comment:
I take it the assumption is if there's an attribute for `SXML_PackageKeyboard_RTL`, then `RTL` is always `true`?

Reply:
Node not attribute, but correct. For a boolean property in XML, the presence of the node signifies it is true. At least in this format. ;-)